src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,178 @@
+package storage;
+
+import task.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.Scanner;
+
+public class Storage {
+
+    private String directory;
+    private String fileName;
+
+    public Storage(String directory, String fileName) {
+        this.directory = directory;
+        this.fileName = fileName;
+    }
+    /**
+     * Saves the tasklist.
+     * Converts a tasklist into its string representation then saves it in a file
+     *
+     * @param taskList  the tasklist to be saved
+     */
+    public void save(TaskList taskList) {
+
+        try {
+            PrintWriter writer = new PrintWriter(this.directory + this.fileName, "UTF-8");
+
+            for (Task task : taskList.getList()) {
+                writer.println(task.tokenize());
+            }
+
+            writer.close();
+
+        } catch (FileNotFoundException | UnsupportedEncodingException e) {
+            e.printStackTrace();
+        }
+
+    }
+    /**
+     * Loads a text file and parses it into a tasklist
+     *
+     * @return
+     */
+    public TaskList load() {
```

Comment:
Similar to the previous comment, perhaps "load" can be renamed to "loadFromHarddisk" or "loadTasklist".

Reply:
Amended to what you have suggested